include/flang/Lower/PFTBuilder.h

```diff
@@ -346,19 +313,19 @@ struct ModuleLikeUnit : public ProgramUnit {
                    const parser::Statement<parser::SubmoduleStmt> *,
                    const parser::Statement<parser::EndSubmoduleStmt> *>;
 
-  ModuleLikeUnit(const parser::Module &m, const ParentType &parent);
-  ModuleLikeUnit(const parser::Submodule &m, const ParentType &parent);
+  ModuleLikeUnit(const parser::Module &m, const ParentType &parentType);
+  ModuleLikeUnit(const parser::Submodule &m, const ParentType &parentType);
   ~ModuleLikeUnit() = default;
   ModuleLikeUnit(ModuleLikeUnit &&) = default;
   ModuleLikeUnit(const ModuleLikeUnit &) = delete;
 
   ModuleStatement beginStmt;
   ModuleStatement endStmt;
-  std::list<FunctionLikeUnit> funcs;
+  std::list<FunctionLikeUnit> containedFunctions;
```

Comment:
Nit: "contained" adds no information IMHO. Everything held by a FunctionLikeUnit is contained in its scope.

Reply:
True, but .. as with the Evaluation --> eval --> e group of naming options, I found it confusing to see so many variations and specialized/overloaded uses of func, funcs, and function.  "funcs" was the most odd case, with relatively few uses, and most amenable to improvement with a longer name.  Maybe nestedFunctions?

Reply:
Switched to nestedFunctions.  That may not be the issue at hand, but it's a little shorter.